<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstWindow</class>
 <widget class="QMainWindow" name="firstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadySet Volunteer - Welcome</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget #centralwidget, #statusbar, #menubar{
	background-color: qlineargradient(spread:pad, x1:0.515633, y1:0.192, x2:0.502367, y2:0.708, stop:0.222707 rgba(21, 21, 21, 255), stop:0.799127 rgba(0, 0, 0, 251));
}

QFrame #logoQFrame{
	background-color: transparent;
    border: none;
	image: url(:/icons/rosewood-logo-red.png);
}
</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame #frame{
	background-color: qlineargradient(spread:pad, x1:0.458934, y1:0.857, x2:0.459035, y2:0.266, stop:0.161572 rgba(214, 0, 0, 255), stop:0.427948 rgba(196, 0, 0, 255), stop:0.68559 rgba(166, 0, 0, 255), stop:0.965066 rgba(147, 0, 0, 255));

}

QToolButton{
   background-color: transparent;
   border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="loginLogoutqToolButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>Login
Logout</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/volunteer64.png</normaloff>:/icons/volunteer64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="projectMngmntQToolButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>Project
Management
</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/edit64.png</normaloff>:/icons/edit64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="logoQFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help </string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/right24.png</normaloff>:/icons/right24.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/info24.png</normaloff>:/icons/info24.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
